package.json

```diff
@@ -685,7 +697,239 @@
           "when": "swift.hasPackage"
         }
       ]
-    }
+    },
+    "breakpoints": [
+      {
+        "language": "arm"
+      },
+      {
+        "language": "asm"
+      },
+      {
+        "language": "c"
+      },
+      {
+        "language": "cpp"
+      },
+      {
+        "language": "objective-c"
+      },
+      {
+        "language": "objective-cpp"
+      },
+      {
+        "language": "rust"
+      },
+      {
+        "language": "swift"
+      }
+    ],
+    "debuggers": [
+      {
+        "type": "swift-lldb",
+        "label": "Native Swift LLDB Debugger",
```

Comment:
I'm not sure about the title, it seems a bit confusing.

Reply:
I'll get rid of the native. I had just added Swift to the original plugin name used